docs/src/get-started/tutorials/get-started.py

```diff
@@ -0,0 +1,148 @@
+"""
```

Comment:
could you add these files to the Python lint path?

Reply:
sure, but we might thought about moving them to `python/tutorials`, which would be consistent with the examples folder. However, then I am not sure how to cleverly handle the dataset withot doing some extensive linking...